src/ActionMonitor/Monitors/AcfMonitor.php

```diff
@@ -2,34 +2,42 @@
 
 namespace WPGatsby\ActionMonitor\Monitors;
 
-class AcfMonitor extends Monitor {
+class AcfMonitor extends Monitor
+{
+    public function init()
+    {
+        // ACF Actions
+        add_action(
+            'acf/update_field_group',
+            function () {
+                $this->trigger_schema_diff(
+                    [
+                        'title' => __('Update ACF Field Group', 'WPGatsby'),
+                    ]
+                );
+            }
+        );
 
-	public function init() {
-		// ACF Actions
-		add_action(
-			'acf/update_field_group',
-			function() {
-				$this->trigger_schema_diff(
-					[
-						'title' => __( 'Update ACF Field Group', 'WPGatsby' ),
-					]
-				);
-			}
-		);
+        add_action(
+            'acf/delete_field_group',
+            function () {
+                $this->trigger_schema_diff(
+                    [
+                        'title' => __('Delete ACF Field Group', 'WPGatsby'),
+                    ]
+                );
+            }
+        );
 
-		add_action(
-			'acf/delete_field_group',
-			function() {
+        add_action(
+			'acf/save_post',
```

Comment:
This will trigger on every `save_post`. This means on every type of post update this will be triggered in addition to ActionMonitors already in place.